README.md

```diff
@@ -54,8 +57,9 @@ Shaderc into.
 ```sh
 git clone https://github.com/google/shaderc $SOURCE_DIR
 cd $SOURCE_DIR/third_party
-git clone https://github.com/google/googletest.git
-git clone https://github.com/google/glslang glslang
+git clone https://github.com/google/googletest.git googletest
+git clone https://github.com/google/glslang.git glslang
+git clone https://github.com/KhronosGroup/SPIRV-Tools.git third_party/spirv-tools
```

Comment:
Good catch:  I don't think we need the last arguments glslang, googletest.
But I think we do need the last argument for spirv-tools since the case is different between the repo name and the local directory name.